javaslang/src/test/java/javaslang/collection/AbstractTraversableRangeTest.java

```diff
@@ -547,29 +547,4 @@ public void shouldProhibitDoubleRangeByFromEqualNaN() {
     public void shouldProhibitDoubleRangeByStepEqualNaN() {
         rangeBy(0.0, 10.0, Double.NaN);
     }
-
```

Comment:
Moved to `Iterator`, because now iteration is possible for elements where the `ulp > step` (i.e. when `i == i+1`)